.github/workflows/main.yaml

```diff
@@ -16,24 +16,23 @@ jobs:
         with:
           fetch-depth: 1
       - name: Setup Ruby
-        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
+        uses: ruby/setup-ruby@97e35c5302afcf3f5ac1df3fca9343d32536b286 # v1.184.0
         with:
-          ruby-version: '3.1' # Not needed with a .ruby-version file
+          ruby-version: '.ruby-version' # Not needed with a .ruby-version file
           bundler-cache: true # runs 'bundle install' and caches installed gems automatically
           cache-version: 0 # Increment this number if you need to re-download cached gems
-      - name: Make sure node is installed.
-        run: sudo apt-get update && sudo apt-get -y install nodejs
-      - name: Install md-to-pdf (NodeJS)
-        run: npm i -g md-to-pdf
       - name: Generate PDF of Tietosuojaseloste
```

Comment:
This just seemed needlessly complicated to me.

Reply:
https://deploy-preview-18--stupefied-roentgen-b689f0.netlify.app/tietosuojaseloste.pdf, doesn't appear as it should.

Reply:
It shows 404 for all PRs

Reply:
Yes, because the workflow for main does the copy, thus the config change did not copy it.

The build directory also doesn't have the PDF as can be seen from netlify's UI.

Reply:
![Screenshot 2024-07-07 at 9 26 19](https://github.com/pikaviestin/website-www.pikaviestin.fi/assets/1703002/c032ddf2-c1a9-4b5e-bb22-cc22264594ff)

Reply:
Netflify uses build command of `bundle exec jekyll build`